### DISCUSSION TITLE
should we try to check other way for replacing strings?

### DISCUSSION THREAD

**Discussion by atherdon** [LLazyEmail/markdown-to-email#751] [Ideas]
https://user-images.githubusercontent.com/1469198/137886892-517a1537-53df-440e-92b7-b661ff621fe5.mp4

**Comment by atherdon** [comment]
we can explore[ readme file](https://github.com/atherdon/markdown-to-email#remark-plugins-to-try-later-later-later)

or just try if remarkjs parser doing it better than we are: https://github.com/remarkjs/remark/tree/main/packages/remark-parse
because they also have a thing, related to markdown->react conversion

**Reply by atherdon** [reply]
сс @vadim9999

**Reply by vadim9999** [reply]
I will check remarkjs
